### DISCUSSION TITLE
picture for inventorypool - i didn't find this feature anymore....

### DISCUSSION THREAD

**Discussion by winampdevil** [leihs/leihs#1476] [Q&A / Community Support] [answered]
Hello, 
i'll create a new inventory-pool and will also use an image for this. In the past (leihs v5), there was an option to upload a picture. 
Now i'm with leihs v6.4.1 - and i didn't find this option anymore. The current inventory-pools have still their own pictures, but i didn't find a way to change these pics.
Any ideas?

Thanks 
Frank

**Comment by nadjaweisskopf** [comment]
@winampdevil Do you mean the image shown in borrow section when the student browses through the categories?

**Comment by winampdevil** [comment, marked as answer]
... i've found it in "Inventory Pools -> Inventory -> Categories"
Thank you for the "category"-hint
